docs/wmco-prerequisites.md

```diff
@@ -2,8 +2,15 @@
 Below are the Cluster and OS prerequisites for WMCO. Please also see the vSphere-specific section that 
 may be relevant.
 
-## Supported Cloud Providers based on OKD/OCP Version and WMCO version
-| Cloud Provider            | Supported OKD/OCP Version | Supported WMCO version |
+## Supported Installation method
+* Installer-Provisioned Infrastructure (IPI) installation method is the recommended and supported installation method.
+
+* User-Provisioned Infrastructure (UPI) is **only supported** on Bare metal or Provider-agnostic
+  platform (`platform: none`) for the [BYOH (Bring Your Own Host)](byoh-instance-pre-requisites.md#byoh-instance-pre-requisites)
+  use case. UPI is not supported in any other platform.
```

Comment:
This is taking about the initial OCP install, right? We support installing Windows nodes into OCP clusters installed by using IPI. We do not support installing Windows nodes into OCP clusters installed by using UPI, except for bare-metal or provider agnostic clusters.  ??

Reply:
Correct, that is accurate.

Reply:
@mburke5678, do you anticipate any rewording to make this message clearer?

Reply:
I can try something if you want.

Reply:
@mburke5678  do you mind dropping a comment with a `/lgtm`  ?